test/unit/foreman/renderer_deprecation_test.rb

```diff
@@ -0,0 +1,31 @@
+require 'test_helper'
+
+class RendererTest < ActiveSupport::TestCase
+  class DummyRenderer
+    attr_accessor :host
+
+    include Foreman::Renderer
+  end
+
+  test 'should show the deprecation for @host.info' do
+    renderer = DummyRenderer.new
+    renderer.host = FactoryGirl.create(:host, :with_puppet)
+    template = FactoryGirl.create(:provisioning_template, :template => '<%= @host.info %>')
+    logger = Rails.logger
+    Rails.logger = mock()
+    Rails.logger.expects(:warn).with("DEPRECATION WARNING: you are using deprecated @host.info in a template, it will be removed in 1.16. Use host_enc instead.").once
+    renderer.unattended_render template
+    Rails.logger = logger
+  end
+
+  test 'should show the deprecation for @host.params' do
+    renderer = DummyRenderer.new
+    renderer.host = FactoryGirl.create(:host, :with_puppet)
+    template = FactoryGirl.create(:provisioning_template, :template => '<%= @host.params %>')
+    logger = Rails.logger
+    Rails.logger = mock()
```

Comment:
ditto, remove these two lines and the `Rails.logger =` line below, just keep .expects.